Libraries/LibCore/CIODevice.h

```diff
@@ -5,7 +5,7 @@
 #include <LibCore/CObject.h>
 
 class CIODevice : public CObject {
-    C_OBJECT(CIODevice)
+    C_OBJECT_ABSTRACT(CIODevice)
```

Comment:
Why is this needed?

Reply:
Must be because of some warning that clang emits about trying to construct this in `construct()`, even though the template never gets instantiated

Reply:
Exactly, clang complains about that construct() function on an abstract class.

Reply:
Well please do that, and other clang-related changes, in a separate commit with a descriptive commit message :)

Reply:
Done!